graylog2-server/src/main/java/org/graylog2/bootstrap/CmdLineTool.java

```diff
@@ -335,8 +346,8 @@ public void doRun(Level logLevel) {
         startCommand();
     }
 
-    protected PluginLoader getPluginLoader(File pluginDir, ChainingClassLoader classLoader) {
-        return new PluginLoader(pluginDir, classLoader);
+    protected PluginLoader getPluginLoader(PluginLoaderConfig pluginLoaderConfig, ChainingClassLoader classLoader) {
```

Comment:
Changing the signature will break other `CmdLineTool`s like the Forwarder. I think we should keep the original method and add a second method with the new signature.

Reply:
Good catch, thank you! I restored the old method.